jest.config.ts

```diff
@@ -7,14 +7,14 @@ const jestConfig: Config = {
   preset: "ts-jest",
   testEnvironment: "jsdom",
   collectCoverageFrom: [
-    "./common/**/*.{js,jsx,ts,tsx}",
-    "./containers/**/*.{js,jsx,ts,tsx}",
-    "./features/**/*.{js,jsx,ts,tsx}",
-    "./pages/**/*.{js,jsx,ts,tsx}",
-    "./store/**/*.{js,jsx,ts,tsx}",
-    "./styles/**/*.{js,jsx,ts,tsx}",
-    "./themes/**/*.{js,jsx,ts,tsx}",
-    "./utils/**/*.{js,jsx,ts,tsx}",
+    "src/common/**/*.{js,jsx,ts,tsx}",
```

Comment:
this can just become "src/**/*.{js,jsx,ts,tsx}"

Reply:
Intentionally left like that for better flexibility in coverage/exclusion.
Replaced with src/ for better readability.

Reply:
fixed